gradle/java.gradle

```diff
@@ -186,7 +186,7 @@ artifacts {
 
 javadoc {
   destinationDir = file("$buildDir/javadoc")
-  options.addStringOption('Xdoclint:none', '-quiet')
+  options.addStringOption('Xwerror', '-quiet')
```

Comment:
Will this be a issue with existing PRs?

Reply:
If in-flight PRs are merged after this change and the build starts failing as a result of javadocs added by the PR being incorrect, the authors of those PRs should revert and fix the javadocs, or submit a second PR to fix the broken javadocs.